### PR TITLE
gocourse04 hw editor and marks

### PR DESCRIPTION
**Pull request by aignatenko88**
Заздалегідь дякую за фідбек)

### CODE REVIEW

File: gocourse04/hw/editor/main.go [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+	"strings"
+)
+
+func scanner(description string) []string {
+	var text []string
+	scanner := bufio.NewScanner(os.Stdin)
+	fmt.Printf(description)
+	for scanner.Scan() {
+		line := scanner.Text()
+		if line == "" {
```
Comment (arriven) [inline gocourse04/hw/editor/main.go:16]: in theory, this shouldn't happen. unless you have some input that looks like
```
asdsd
asds

kjdshdfkj
```
but in that case I think we should still try to handle it and not break the loop (scanner should do that automatically)

Comment (arriven) [reply gocourse04/hw/editor/main.go:16]: ok, nevermind, I see that you use two different inputs that you are splitting by a newline

File: gocourse04/hw/marks/main.go [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+import (
+    "fmt"
+)
+
+func calcAvgMark (marks []float64) float64 {
+    var sum float64
+    for _, v := range marks {
+        sum += v
+    }
+    return sum / float64(len(marks))
```
Comment (arriven) [inline gocourse04/hw/marks/main.go:17]: what would happen if you pass an empty array here?

Comment (aignatenko88) [reply gocourse04/hw/marks/main.go:17]: буде в результаті ділення Nan)
додав перевірку на пустий slice

**Review by arriven — APPROVED** [review]

**Pull request merged**